app/models/report.rb

```diff
@@ -20,17 +21,25 @@ class Report < ApplicationRecord
 
   def self.inherited(child)
     child.instance_eval do
-      scoped_search :relation => :host,         :on => :name,  :complete_value => true, :rename => :host
+      scoped_search :relation => :host, :on => :name, :complete_value => true, :rename => :host
+      scoped_search :relation => :host_owner,
+        :on => :id,
+        :complete_value => true,
+        :rename => :host_owner_id,
+        :only_explicit => true,
+        :validator => ->(value) { ScopedSearch::Validators::INTEGER.call(value) },
+        :value_translation => ->(value) { value == 'current_user' ? User.current.id : value },
+        :special_values => %w[current_user]
       scoped_search :relation => :organization, :on => :name,  :complete_value => true, :rename => :organization
```

Comment:
You've missed one and now rubocop doesn't like it when it's not aligned with others
```suggestion
      scoped_search :relation => :organization, :on => :name, :complete_value => true, :rename => :organization
```